app/src/NativeBase/Components/Checkbox.tsx

```diff
@@ -0,0 +1,44 @@
+/**
+ * @file CheckBox component for the STA Volunteer App
+ */
+
+import { Pressable } from 'native-base'
+import React from 'react'
+import { Ionicons } from '@expo/vector-icons'
+
+type CheckboxProps = {
+  onChange: (value: string[]) => void
```

Comment:
Why does a checkbox return a string array?  I would have expected each checkbox to just return a string?